### ISSUE TITLE
Styles broken after updating to Bootstrap 4

### ISSUE THREAD

**Issue by hmurt** [softsimon/angular-2-dropdown-multiselect#379]
After updating from Bootstrap 3.3.7 to 4.0.0, dropdown styling is broken and looks weird:
![image](https://user-images.githubusercontent.com/15627072/35852650-1f5d64c8-0b34-11e8-893f-58253ce6f5b3.png)

I'm using ngx-bootstrap 2.0.2.

**Comment by eappell** [comment]
I'm having the same issue.  I've tried setting an itemClass, but it's not applied at all, as far as I can tell.  It appears that Bootstrap's reboot.scss is setting the color of the anchor tag to the same color as the highlight for the active item.  I'm just about to deploy this, but I can't until this is resolved...

**Comment by Ezeon** [comment]
This can be fixed with CSS.

Solution from #335 
```
ss-multiselect-dropdown {
  & .dropdown-item.active,
  & .dropdown-item:active {
    background-color: $myColor;
  }
}
```

**Comment by dagonar** [comment]
on the form that you are using you can fix this with @Ezeon response, I'll just add my code also.

`:host ::ng-deep ss-multiselect-dropdown {
	padding: 0;

	& .dropdown {
		width: 100%;

		> ul.dropdown-menu {
			padding: initial 5px;
			border-radius: 5px;
			width: 100%;
		}
	}

	& li.dropdown-item.active > a,
	& li.dropdown-item:active > a {
		color: white;
	}

	& .input-group-addon {
		padding: 5px;
	}
}`

**Comment by tom-riddle** [comment]
how to fix this in normal css?

**Comment by softsimon** [comment]
I will soon post a new version with default Bootstrap 4 support.

**Comment by tom-riddle** [comment]
Here is a fix using normal css : 
:host ::ng-deep ss-multiselect-dropdown li.dropdown-item.active > a, :host ::ng-deep ss-multiselect-dropdown li.dropdown-item:active > a {
    color: white;
  }

**Comment by maxsands1503** [comment]
@tom-riddle does your CSS fix work for v1.8? I just upgraded and I the multiselect dropdown styling broke.
